src/components/product/ProductCard/product-card.scss

```diff
@@ -49,11 +49,23 @@
   }
 
   [data-card-content] a[href] {
+    display: -webkit-box;
```

Comment:
I don't think we need to apply those prefixes, it's added automatically 🤔  `-webkit`, `-moz` ...  Can you double check if it's working?

Reply:
This value is required for the `line-clamp`. 
When I was testing it, I notice that it only works with the `-webkit` prefix.
I think the `autoprefixer` is not being able to add the prefixes for `box` value. Maybe because is an old draft of the new flexbox. But there is no solution using flexbox, though.

There is this warning on [autoprefixer github](https://github.com/postcss/autoprefixer#warnings). So I decided to keep it so far. 
I tested using Chrome, Sarafi, and Firefox.

Any suggestion?

Reply:
Oh! Got it! So let's keep it :)